tests/testdata/envoy_local_v2.yaml

```diff
@@ -0,0 +1,23 @@
+admin:
```

Comment:
Can you create a subdir - it may confuse the other 'file-based' adapter which reads from this directory.

Reply:
done